storybook/.storybook/theme.ts

```diff
@@ -0,0 +1,47 @@
+import { create } from '@storybook/theming/create';
+
+export const cocoTheme = create({
+  // Mode
+  base: 'light',
+
+  // Brand
+  brandTitle: 'commercetools ui-kit',
+  brandUrl: './',
+  brandImage:
+    'https://unpkg.com/@commercetools-frontend/assets/logos/commercetools_primary-logo_horizontal_RGB.png',
+  brandTarget: '_self',
+
+  // Typography
+  fontBase: '"Inter", serif',
+  fontCode: 'monospace',
+
+  //
+  // colorPrimary: '#3A10E5',
+  // colorSecondary: '#585C6D',
+
+  colorPrimary: 'hsl(240, 64%, 58%)',
+  colorSecondary: 'hsl(195, 70%, 50%)',
```

Comment:
At some point it might be worth using token/theme values from the `/design-system` package for the storybook theme for consistency/ease of change management.